camus-sweeper/src/main/java/com/linkedin/camus/sweeper/CamusSweeper.java

```diff
@@ -222,8 +226,24 @@ public void run() throws Exception
       {
         System.err.println("unable to process " + topicName + " skipping...");
         e.printStackTrace();
-        executorService.shutdown();
-        throw e;
```

Comment:
It's a bit hard to simulate "means unable to create a task/future". This is how Camus worked originally, I removed whole `try`/`catch` block and with a simple `Exception` it now exits with `1`.